src/main/java/dev/aziz/movies/services/PeopleService.java

```diff
@@ -0,0 +1,84 @@
+package dev.aziz.movies.services;
+
+import dev.aziz.movies.dtos.ActorDto;
+import dev.aziz.movies.dtos.DirectorDto;
+import dev.aziz.movies.entities.Actor;
+import dev.aziz.movies.entities.Director;
+import dev.aziz.movies.entities.Genre;
+import dev.aziz.movies.entities.Movie;
+import dev.aziz.movies.entities.People;
+import dev.aziz.movies.exceptions.AppException;
+import dev.aziz.movies.mappers.MovieMapper;
+import dev.aziz.movies.repositories.ActorRepository;
+import dev.aziz.movies.repositories.DirectorRepository;
+import dev.aziz.movies.repositories.MovieRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Service
+public class PeopleService {
+
+    private final ActorRepository actorRepository;
+    private final DirectorRepository directorRepository;
+    private final MovieRepository movieRepository;
+    private final MovieMapper movieMapper;
+
+
+    public List<String> getAllPeople() {
+        List<String> listOfActors = actorRepository.findAll().stream()
+                .map(actor -> actor.getFirstName() + " " + actor.getLastName())
+                .toList();
+
+        List<String> listOfDirectors = directorRepository.findAll().stream()
+                .map(director -> director.getFirstName() + " " + director.getLastName())
+                .toList();
+        List<String> allPeople = new ArrayList<>();
+
+        allPeople.addAll(listOfActors);
+        allPeople.addAll(listOfDirectors);
+        return allPeople;
+    }
+
+
+    public Object getOnePerson(String type, Long id) {
+        if (People.ACTOR.toString().equals(type.toUpperCase())) {
```

Comment:
You can use the enum this way, but are able to map the type People directly in the controller's input argument, in the PathVariable?